modules/labs-react/common/stories/Box.stories.mdx

```diff
@@ -0,0 +1,404 @@
+Style props are special design-system-aware props that allow you to use Canvas design tokens
+efficiently as you build components. Style props are better understood by their categories, which
+are listed below. Each category has a more detailed explanation in the following subsections.
+
+### Categories
+
+- [Border](#border)
+  - [Border Logical Props](#border-logical-props)
+  - [Border Shorthand Props](#border-shorthand-props)
+  - [Border Color Props](#border-color-props)
+  - [Border Radius Props](#border-radius-props)
+  - [Border Line Style Props](#border-line-style-props)
+  - [Border Width Props](#border-width-props)
+- [Color](#color)
+- [Depth](#depth)
+- [Flex Item](#flex-item)
+- [Layout](#layout)
+- [Position](#position)
+  - [Position Logical Props](#position-logical-props) =
+    [Position Standard Props](#position-standard-props)
+- [Space](#space)
+  - [Space Logical Props](#space-logical-props)
+  - [Space Standard Props](#space-standard-props)
+
+### Border
+
+Border style props allow you to adjust the border styles of components. Some props, such as
+`borderRadius` and `borderColor`, are connected to our design tokens. This makes it really simple to
+use Canvas values as you build UI. There are five types of border props:
+`BorderShorthandStyleProps`, `BorderColorStyleProps`, `BorderRadiusStyleProps`,
+`BorderLineStyleProps`, and `BoxBorderWidthStyleProps`.
+
+#### Border Logical Props
+
+Border logical props are based on
+[CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts).
+They support bidirectionality (LTR and RTL) and will automatically flip the value when a change in
+direction is detected. For example: `borderInlineStart="1px solid #000"` would set
+`border-left: 1px solid #000;` for LTR and switch to `border-right: 1px solid #000;` for RTL. You'll
+want to opt for these properties over standard border props, such as `borderLeft` and `borderRight`
+when you need bidrectional styles.
+
+Here's a quick reference for how these props map to LTR and RTL styles:
+
+| Logical Prop                           | LTR Style                            | RTL Style                            |
+| -------------------------------------- | ------------------------------------ | ------------------------------------ |
+| `borderInlineStart`                    | `border-left`                        | `border-right`                       |
+| `borderInlineEnd`                      | `border-right`                       | `border-left`                        |
+| `borderInlineStart{Color,Style,Width}` | `border-left{-color,-style,-width}`  | `border-right{-color,-style,-width}` |
+| `borderInlineEnd{Color,Style,Width}`   | `border-right{-color,-style,-width}` | `border-left{-color,-style,-width}`  |
+
+And below is a props table for the logical props:
+
+<ArgsTable of={BoxBorderLogical} />
+
+#### Border Shorthand Props
+
+These props map to CSS border shorthand properties:
```

Comment:
Nit: OCD kicking in here. Don't think the "CSS" is necessary here, especially given that it's not used elsewhere in similar phrases throughout the document.

```suggestion
These props map to border shorthand properties:
```